controllers/jobs.js

```diff
@@ -0,0 +1,31 @@
+const Job = require('../models/Job')
+const {StatusCodes} = require('http-status-codes')
+const {BadRequestError, NotFoundError} = require('../errors')
+
+const getAllJobs = async (req, res) => {
+    res.send('get all jobs')
```

Comment:
I think you forgot to fill this out. The video segment for week9 shows how to create this

Reply:
Hello, thank you so much for reviewing my assignments and providing your feedback!
 I re-watched the video lesson, and unfortunately, I do not understand where I need to make changes. Based on the task assignment from CTD for 'Week 8 of 17', it states, 'Before you start this lesson, you create the week9 git branch.' So, the week9 branch is actually for week 8 of lessons. This lesson runs from 6:28:35 to 8:20:35. I have attached a screenshot from the video at 8:20.
<img width="990" alt="Screen Shot 2024-05-17 at 12 49 04 PM" src="https://github.com/ElenaDu/06-jobs-api/assets/35644813/ee0cf39e-1d5a-4900-b079-860644cee9e5">                                                                               I changed the comment in my pull request since it probably was misleading. I would appreciate more information if I did not understand the issue here and need to correct.

Reply:
I thought that was strange when I saw it. My bad, I have been out of commission for a few days. Resubmit and I will approve it immediately, as all else looked good.